acid/features/history/tests/conftest.py

```diff
@@ -1,23 +1,80 @@
 # -*- coding: utf-8 -*-
-from datetime import datetime
+import datetime
 
 import pytest
 
 from pony.orm import commit, db_session
 
 from ..model import ZuulBuild, ZuulBuildSet
 
+TIME_START = '2018-02-23 22:00:00'
+TIME_END = '2018-02-23 23:55:00'
 
-@pytest.fixture
-def zuul_builds(mocker):
-    mocker.patch('pony.orm.core.commit')
-    return [make_build(build_number=104, buildset_id=5010),
-            make_build(build_number=104, buildset_id=5010)]
+
+def date_to_epoch(t_stamp):
```

Comment:
1. Epoch is not a good name here. Epoch, for the purpose of date calculation in computer systems, is just a year - and this year number marks a 'beginning of time' for a computer system.
2. 'date' in the function name does not really indicate that the argument is intended to be a datetime object. Simply 'date' may suggest just a date formatted string.

What we do here, is converting a datetime object to number of seconds since that Epoch year.
More fitting function name would be: "datetime_to_seconds"

Further naming convention should be changed - substitute 'epoch' for 'seconds', and 'date' to 'datetime' if you are dealing with datetime objects.